incubator/hnc/test/e2e/demo_test.go

```diff
@@ -0,0 +1,64 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "sigs.k8s.io/multi-tenancy/incubator/hnc/pkg"
+)
+
+var _ = Describe("Subnamespaces", func() {
```

Comment:
s/Subnamespaces/Demo/

Reply:
Also add a comment pointing to the demo doc, and (once this is approved) add a warning to the top of the demo doc asking authors to keep it in sync with this file.

Reply:
done